apps/gitness/src/pages/pull-request/pull-request-conversation-page.tsx

```diff
@@ -0,0 +1,338 @@
+import { useCallback, useEffect, useState } from 'react'
+import { Spacer } from '@harnessio/canary'
+import {
+  commentCreatePullReq,
+  EnumCheckStatus,
+  EnumMergeMethod,
+  mergePullReqOp,
+  OpenapiMergePullReq,
+  TypesPullReqActivity,
+  TypesPullReqReviewer,
+  useCodeownersPullReqQuery,
+  useListPullReqActivitiesQuery,
+  useReviewerListPullReqQuery
+} from '@harnessio/code-service-client'
+import {
+  FullWidth2ColumnLayout,
+  PullRequestCommentBox,
+  PullRequestFilters,
+  PullRequestOverview,
+  PullRequestPanel,
+  PullRequestSideBar,
+  SkeletonList
+} from '@harnessio/playground'
+import { useDateFilters } from './hooks/useDataFilters'
+import { useActivityFilters } from './hooks/useActivityFilters'
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import {
+  capitalizeFirstLetter,
+  checkIfOutdatedSha,
+  extractInfoForCodeOwnerContent,
+  findChangeReqDecisions,
+  findWaitingDecisions,
+  processReviewDecision
+} from './utils'
+import { useParams } from 'react-router-dom'
+import { PathParams } from '../../RouteDefinitions'
+import { usePullRequestData } from './context/pull-request-data-provider'
+import { isEmpty } from 'lodash-es'
+import { CodeOwnerReqDecision } from '../../types'
+export const diffdata = {
```

Comment:
Is this mock data?

Reply:
It’s mock data cause I was still working on adding the api for diff data